### PR TITLE
Update wip-learning.md

### PR DESCRIPTION
**Pull request by nenonix**
Removed mention of OMS and added reference and links to Device Health (First Draft) https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3236

### CODE REVIEW

**Comment by mypil** [comment]
@Justinha - Can you please assist to check, approve and merge the content for this PR based on issue #3236?

Thank you.

**Comment by nenonix** [comment]
@Justinha  done, thanks

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/information-protection/windows-information-protection/wip-learning.md [hunk 1]
```diff
@@ -44,59 +44,42 @@ In the **Website learning report**, you can view a summary of the devices that h
...
     ![View of Add Apps app info entry boxes](images/wip-learning-app-info.png)
 
-7. Type the name of the product in **PRODUCT NAME** (required)  (this will probably be the same as what you typed for **NAME**).
+6. Type the name of the product in **PRODUCT NAME** (required)  (this will probably be the same as what you typed for **NAME**).
 
-8. Back in OMS, copy the name of the executable (for example, snippingtool.exe) and then go back to Intune and paste it in **FILE** (required).
+7. Copy the name of the executable (for example, snippingtool.exe) and paste it in **FILE** (required).
 
-9. Go back to OMS one more time and note the version number of the app and type it in **MIN VERSION** in Intune (alternately, you can specify the max version, but one or the other is required), and then select the **ACTION**: **Allow** or **Deny**
+8. Tyoe the version number of the app into **MIN VERSION** in Intune (alternately, you can specify the max version, but one or the other is required), and then select the **ACTION**: **Allow** or **Deny**
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/wip-learning.md]:
```suggestion
8. Type the version number of the app into **MIN VERSION** in Intune (alternately, you can specify the max version, but one or the other is required), and then select the **ACTION**: **Allow** or **Deny**
```

**Comment by mypil** [comment]
@Justinha - Kindly check, approve and merge the content for this PR based on issue #3236?

Thank you.

**Pull request merged**